example/client/agent/go.mod

```diff
@@ -0,0 +1,11 @@
+module agent
```

Comment:
please remove the `example/client/agent/agent ` binary file on this branch.
do we need go.mod and go.sum here?

Reply:
thank you for your review.
I think go.mod & go.sum need.
Could you please tell me the reason that you want to delete?

Reply:
:memo:
I discussed with oral. and It was concluded that `go.mod go.sum` is necessary.